src/services/pscVerificationService.ts

```diff
@@ -79,3 +79,26 @@ export const patchPscVerification = async (request: Request, transactionId: stri
 
     return castedSdkResponse;
 };
+
+export const checkPlannedMaintenance = async (request: Request): Promise<ApiResponse<PlannedMaintenance>> => {
+    const apiClient: ApiClient = createApiKeyClient();
+
+    logger.debug(`Checking Planned Maintenance for PSC Verification API service`);
+    const sdkResponse: ApiResponse<PlannedMaintenance> | ApiErrorResponse = await apiClient.pscVerificationService.checkPlannedMaintenance();
+
+    if (!sdkResponse) {
```

Comment:
Test coverage for error code-paths needed

Reply:
added coverage for checkPlannedMaintenance and other existing calls